### DISCUSSION TITLE
Hi, I have installed openDCIM and i want to upload bulk data. But that option is not showing in my login. Is there any admin login to get that option?

### DISCUSSION THREAD

**Discussion by CSKCNU** [opendcim/openDCIM#1426] [Q&A] [answered]
Hi, I have installed openDCIM and i want to upload bulk data. But that option is not showing in my login. Is there any admin login to get that option?

**Comment by samilliken** [comment]
You have to give your UserID that right.

**Reply by CSKCNU** [reply]
Do you mean the right to be given to mysql UserID? If yes, which rights? Kindly help me.

**Comment by timlegge** [comment, marked as answer]
Under that user in opendcim check the box next to bulk load
